Shared/Samples/Show geodesic sector and ellipse/ShowGeodesicSectorAndEllipseView.swift

```diff
@@ -0,0 +1,396 @@
+                    let symbol = SimpleLineSymbol(style: .solid, color: .green, width: 2)
+                    addSectorGraphic(geometry: geometry, symbol: symbol)
+                }
+            case .polygon:
+                // Generate sector as a filled polygon.
+                var parameters = GeodesicSectorParameters<ArcGIS.Polygon>()
+                fillSectorParameters(&parameters)
+                if let geometry = GeometryEngine.geodesicSector(parameters: parameters) {
+                    let symbol = SimpleFillSymbol(style: .solid, color: .green)
+                    addSectorGraphic(geometry: geometry, symbol: symbol)
+                }
+            }
+        }
+        
+        /// Populates a geodesic sector parameters value with current user-defined values.
+        /// - Parameter parameters: A reference to the parameter struct that will be filled.
+        private func fillSectorParameters<T>(_ parameters: inout GeodesicSectorParameters<T>) {
+            parameters.center = center
+            parameters.axisDirection = axisDirection.value
+            parameters.maxPointCount = Int(maxPointCount.rounded())
+            parameters.maxSegmentLength = maxSegmentLength
+            parameters.sectorAngle = sectorAngle.value
+            parameters.semiAxis1Length = semiAxis1Length
+            parameters.semiAxis2Length = semiAxis2Length
+            parameters.startDirection = startDirection
+            parameters.linearUnit = .miles
+        }
+        
+        /// Adds a sector graphic to the overlay and applies the appropriate renderer.
+        private func addSectorGraphic(geometry: Geometry, symbol: Symbol) {
+            let sectorGraphic = Graphic(geometry: geometry, symbol: symbol)
+            sectorGraphicOverlay.addGraphic(sectorGraphic)
+        }
+        
+        /// Generates and adds a geodesic ellipse graphic based on the current settings and center point.
+        private func updateEllipse() {
+            let parameters = GeodesicEllipseParameters<ArcGIS.Polygon>(
+                axisDirection: axisDirection.value,
+                center: center,
+                linearUnit: .miles,
+                maxPointCount: Int(maxPointCount.rounded()),
+                maxSegmentLength: maxSegmentLength,
+                semiAxis1Length: semiAxis1Length,
+                semiAxis2Length: semiAxis2Length
+            )
+            let geometry = GeometryEngine.geodesicEllipse(parameters: parameters)
+            let symbol = SimpleLineSymbol(style: .dash, color: .red, width: 2)
+            let graphic = Graphic(geometry: geometry, symbol: symbol)
+            ellipseGraphicOverlay.addGraphic(graphic)
+        }
+    }
+    
+    struct SectorSettingsView: View {
+        @ObservedObject var model: ShowGeodesicSectorAndEllipseView.Model
+        
+        private var numberFormat: FloatingPointFormatStyle<Double> {
+            .init().precision(.fractionLength(0)).grouping(.never)
```

Comment:
I'm curious why you chose not to display grouping separators? That's a bit unexpected. Consider showing them.

Reply:
I think because of the changes, it's no longer needed. There does not appear to be any ui difference with it removed.